mission-control-client/package.json

```diff
@@ -3,21 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.5.1",
     "axios": "^0.19.0",
+    "formik": "^1.5.8",
     "node-sass": "^4.12.0",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
-    "react-redux": "^7.1.1",
     "react-ga": "^2.7.0",
-    "react-scripts": "3.2.0",
-    "validator": "^11.1.0",
+    "react-redux": "^7.1.1",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
     "redux": "^4.0.4",
     "redux-thunk": "^2.3.0",
     "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^0.88.1",
-    "validator": "^11.1.0"
```

Comment:
Do we need the validator package?

Reply:
No, that was something old that I was using. Yup is taking over for validator.